src/App.js

```diff
@@ -1,45 +1,37 @@
 import { Link } from "react-router-dom";
-import {UserProvider} from "./UserContext"
+
 function App() {
- 
-return ( 
-  <UserProvider>
```

Comment:
UserProvider e state, si trebuie sa inconjoare toata aplicatia, de acee trebuie sa fie in index.js, nu doar in App.js